packages/sdk/examples/create-dapp.ts

```diff
@@ -0,0 +1,26 @@
+import { Dialect, NodeDialectWalletAdapter } from '../src';
+
+async function main() {
+  // const sdk = Dialect.sdk({
```

Comment:
Should we remove the commented out code or is there a special meaning to it?

Reply:
Actually, we don't need this file, deleted